src/App.js

```diff
@@ -35,22 +34,30 @@ function App() {
         yearlyContribution: yearlyContribution,
       });
     }
-
-  
   }
-    // do something with yearlyData ...
-    // console.log(yearlyData[0]);
+  // do something with yearlyData ...
+  // console.log(yearlyData[0]);
 
   return (
     <div>
       <Header />
 
-      <Form userInputCalculator={calculateHandler} />
-
-  {!userInput && <center>please input data for calculation</center>}
-  {userInput && <Table calculatedData={yearlyData} initialInvestment={userInput["current-savings"]}/>}
+      <Form userInputCalculator={calculateHandler} 
+        setUserInput={setUserInput}
+      />
 
-      
+      {!userInput && <>
```

Comment:
Here you could have used ternary operator

Reply:
> Here you could have used ternary operator

ternary operator applied, kindly verify.